### ISSUE TITLE
HTML/Text Extractor

### ISSUE THREAD

**Issue by bwhiteman** [Sotera/DatawakeDepot#152]
We should have a simple extractor that pulls the HTML and extracted body text of a document.

**Comment by bmcdougald** [comment]
@michaelsframe don't we already have this via the StanNER extractor?  I'm also getting body content persisted and viewable in the trail url section

**Comment by michaelsframe** [comment]
I'm not sure what this is asking. The current trailing process pulls and sends the <body> element of the page to the extractor.

What do you mean by document?

**Comment by bwhiteman** [comment]
Does it persist the HTML and the plain text "body"?

For future analytics, it would be good to have an extractor that pulls the main body text with something like unfluff and returns it. I had the stanbol extractor doing it but it shouldn't be there.

Once this is done, we can run text analytics on the corpus of pages that has been scraped.

**Comment by michaelsframe** [comment]
If we have the HTML it seems redundant to store the plain text body, doubles storage requirements and will slow down the URL insertion.  Any analytic can retrieve and unfluff as necessary right?

**Comment by bwhiteman** [comment]
Yes, this is a question of which is more costly, to write text once read it may times or to extract the text every time.
